llvm/lib/IR/Globals.cpp

```diff
@@ -110,6 +110,10 @@ bool GlobalValue::isInterposable() const {
 }
 
 bool GlobalValue::canBenefitFromLocalAlias() const {
+  if (isTagged()) {
+    // Cannot create local aliases to MTE tagged globals.
```

Comment:
Consider extending the comment to explain why (because the address of an MTE global includes a tag that is assigned by the dynamic loader in GOT).